spec/controllers/criteria_controller_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe CriteriaController, type: :controller do
 
-  describe 'Using Flexible criteria' do
+  describe 'Using Rubric criteria' do
```

Comment:
I have two blocks, one for rubric criteria and another one for flexible criteria. Rubric Criteria is usually first, so I just swapped the tests. However, they are very similar, so git interpreted the swap this way. I ran the tests and it is working fine.